src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```
@@ -461,6 +462,31 @@ This package is an internal implementation of the .NET Core SDK and is not meant
           StandardOutputImportance="High" />
   </Target>
 
+  <Target Name="_GenerateComposites"
+    DependsOnTargets="_ExpandRuntimePackageRoot">
+
+    <PropertyGroup>
+      <CompositeTargetDir>$(TargetDir)Composite\</CompositeTargetDir>
+      <ManagedAssetsFullPath>$(RuntimePackageRoot)$(ManagedAssetsPackagePath)</ManagedAssetsFullPath>
+      <NativeAssetsFullPath>$(RuntimePackageRoot)$(NativeAssetsPackagePath)</NativeAssetsFullPath>
+
+      <BaseAssemblies>$(ManagedAssetsFullPath)\*.dll</BaseAssemblies>
+      <BaseAssemblies Condition="'Exists($(ManagedAssetsFullPath))' == 'true' and '$(TargetOsName)' == 'win'">$(BaseAssemblies) $(NativeAssetsFullPath)\*.dll</BaseAssemblies>
```

suggest
```suggestion
      <BaseAssemblies Condition="Exists('$(NativeAssetsFullPath)') and '$(TargetOsName)' == 'win'">$(BaseAssemblies) $(NativeAssetsFullPath)\*.dll</BaseAssemblies>
```

I'm also not sure you need both `Exists(...)` and the `$(TargetOsName)` checks; may be redundant (especially given that things worked when the `Exists(...)` check was always `true`). I prefer `Exists(...)` because it's more future proof, both for new native assets on some non-Windows platforms and for deciding not to ship native assets on some Windows platforms.